From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Richard Townsend <Richard.Townsend@arm.com>
Date: Fri, 29 May 2020 20:14:25 +0000
Subject: rebaseline ichromeaccessible for Windows arm64

To rebaseline: copied x64 => arm64, ninja, executed suggested copy
command.

Change-Id: Ibf2e9f9d93d10c3f2946d5890c66e2956c576baf
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2222349
Commit-Queue: Richard Townsend <richard.townsend@arm.com>
Commit-Queue: Nico Weber <thakis@chromium.org>
Reviewed-by: Nico Weber <thakis@chromium.org>
Cr-Commit-Position: refs/heads/master@{#773310}

diff --git a/third_party/win_build_output/midl/ui/accessibility/platform/arm64/ichromeaccessible.dlldata.c b/third_party/win_build_output/midl/ui/accessibility/platform/arm64/ichromeaccessible.dlldata.c
new file mode 100644
index 0000000000000000000000000000000000000000..3ee5c29ef3d46b331af651366063725e48452e51
--- /dev/null
+++ b/third_party/win_build_output/midl/ui/accessibility/platform/arm64/ichromeaccessible.dlldata.c
@@ -0,0 +1,37 @@
+/*********************************************************
+   DllData file -- generated by MIDL compiler 
+
+        DO NOT ALTER THIS FILE
+
+   This file is regenerated by MIDL on every IDL file compile.
+
+   To completely reconstruct this file, delete it and rerun MIDL
+   on all the IDL files in this DLL, specifying this file for the
+   /dlldata command line option
+
+*********************************************************/
+
+
+#include <rpcproxy.h>
+
+#ifdef __cplusplus
+extern "C"   {
+#endif
+
+EXTERN_PROXY_FILE( ichromeaccessible )
+
+
+PROXYFILE_LIST_START
+/* Start of list */
+  REFERENCE_PROXY_FILE( ichromeaccessible ),
+/* End of list */
+PROXYFILE_LIST_END
+
+
+DLLDATA_ROUTINES( aProxyFileList, GET_DLL_CLSID )
+
+#ifdef __cplusplus
+}  /*extern "C" */
+#endif
+
+/* end of generated dlldata file */
diff --git a/third_party/win_build_output/midl/ui/accessibility/platform/arm64/ichromeaccessible.h b/third_party/win_build_output/midl/ui/accessibility/platform/arm64/ichromeaccessible.h
new file mode 100644
index 0000000000000000000000000000000000000000..c88b869e031b4179cf1c6dd0a742af93c0e2c50e
--- /dev/null
+++ b/third_party/win_build_output/midl/ui/accessibility/platform/arm64/ichromeaccessible.h
@@ -0,0 +1,296 @@
+
+
+/* this ALWAYS GENERATED file contains the definitions for the interfaces */
+
+
+ /* File created by MIDL compiler version 8.xx.xxxx */
+/* at a redacted point in time
+ */
+/* Compiler settings for ../../ui/accessibility/platform/ichromeaccessible.idl:
+    Oicf, W1, Zp8, env=Win64 (32b run), target_arch=ARM64 8.01.0622 
+    protocol : dce , ms_ext, c_ext, robust
+    error checks: allocation ref bounds_check enum stub_data 
+    VC __declspec() decoration level: 
+         __declspec(uuid()), __declspec(selectany), __declspec(novtable)
+         DECLSPEC_UUID(), MIDL_INTERFACE()
+*/
+/* @@MIDL_FILE_HEADING(  ) */
+
+#pragma warning( disable: 4049 )  /* more than 64k source lines */
+
+
+/* verify that the <rpcndr.h> version is high enough to compile this file*/
+#ifndef __REQUIRED_RPCNDR_H_VERSION__
+#define __REQUIRED_RPCNDR_H_VERSION__ 475
+#endif
+
+#include "rpc.h"
+#include "rpcndr.h"
+
+#ifndef __RPCNDR_H_VERSION__
+#error this stub requires an updated version of <rpcndr.h>
+#endif /* __RPCNDR_H_VERSION__ */
+
+#ifndef COM_NO_WINDOWS_H
+#include "windows.h"
+#include "ole2.h"
+#endif /*COM_NO_WINDOWS_H*/
+
+#ifndef __ichromeaccessible_h__
+#define __ichromeaccessible_h__
+
+#if defined(_MSC_VER) && (_MSC_VER >= 1020)
+#pragma once
+#endif
+
+/* Forward Declarations */ 
+
+#ifndef __IChromeAccessibleDelegate_FWD_DEFINED__
+#define __IChromeAccessibleDelegate_FWD_DEFINED__
+typedef interface IChromeAccessibleDelegate IChromeAccessibleDelegate;
+
+#endif 	/* __IChromeAccessibleDelegate_FWD_DEFINED__ */
+
+
+#ifndef __IChromeAccessible_FWD_DEFINED__
+#define __IChromeAccessible_FWD_DEFINED__
+typedef interface IChromeAccessible IChromeAccessible;
+
+#endif 	/* __IChromeAccessible_FWD_DEFINED__ */
+
+
+/* header files for imported files */
+#include "objidl.h"
+#include "oaidl.h"
+
+#ifdef __cplusplus
+extern "C"{
+#endif 
+
+
+/* interface __MIDL_itf_ichromeaccessible_0000_0000 */
+/* [local] */ 
+
+#define	DISPID_CHROME_BULK_FETCH	( -1600 )
+
+#define	DISPID_CHROME_ON_BULK_FETCH_RESULT	( -1601 )
+
+#define	DISPID_CHROME_HIT_TEST	( -1602 )
+
+#define	DISPID_CHROME_ON_HIT_TEST_RESULT	( -1603 )
+
+
+
+extern RPC_IF_HANDLE __MIDL_itf_ichromeaccessible_0000_0000_v0_0_c_ifspec;
+extern RPC_IF_HANDLE __MIDL_itf_ichromeaccessible_0000_0000_v0_0_s_ifspec;
+
+#ifndef __IChromeAccessibleDelegate_INTERFACE_DEFINED__
+#define __IChromeAccessibleDelegate_INTERFACE_DEFINED__
+
+/* interface IChromeAccessibleDelegate */
+/* [unique][uuid][object] */ 
+
+
+EXTERN_C const IID IID_IChromeAccessibleDelegate;
+
+#if defined(__cplusplus) && !defined(CINTERFACE)
+    
+    MIDL_INTERFACE("0e3edc14-79f4-413f-b854-d3b6860d74a2")
+    IChromeAccessibleDelegate : public IUnknown
+    {
+    public:
+        virtual /* [id][propput] */ HRESULT STDMETHODCALLTYPE put_bulkFetchResult( 
+            /* [in] */ LONG requestID,
+            /* [in] */ BSTR resultJson) = 0;
+        
+        virtual /* [id][propput] */ HRESULT STDMETHODCALLTYPE put_hitTestResult( 
+            /* [in] */ LONG requestID,
+            /* [in] */ IUnknown *result) = 0;
+        
+    };
+    
+    
+#else 	/* C style interface */
+
+    typedef struct IChromeAccessibleDelegateVtbl
+    {
+        BEGIN_INTERFACE
+        
+        HRESULT ( STDMETHODCALLTYPE *QueryInterface )( 
+            IChromeAccessibleDelegate * This,
+            /* [in] */ REFIID riid,
+            /* [annotation][iid_is][out] */ 
+            _COM_Outptr_  void **ppvObject);
+        
+        ULONG ( STDMETHODCALLTYPE *AddRef )( 
+            IChromeAccessibleDelegate * This);
+        
+        ULONG ( STDMETHODCALLTYPE *Release )( 
+            IChromeAccessibleDelegate * This);
+        
+        /* [id][propput] */ HRESULT ( STDMETHODCALLTYPE *put_bulkFetchResult )( 
+            IChromeAccessibleDelegate * This,
+            /* [in] */ LONG requestID,
+            /* [in] */ BSTR resultJson);
+        
+        /* [id][propput] */ HRESULT ( STDMETHODCALLTYPE *put_hitTestResult )( 
+            IChromeAccessibleDelegate * This,
+            /* [in] */ LONG requestID,
+            /* [in] */ IUnknown *result);
+        
+        END_INTERFACE
+    } IChromeAccessibleDelegateVtbl;
+
+    interface IChromeAccessibleDelegate
+    {
+        CONST_VTBL struct IChromeAccessibleDelegateVtbl *lpVtbl;
+    };
+
+    
+
+#ifdef COBJMACROS
+
+
+#define IChromeAccessibleDelegate_QueryInterface(This,riid,ppvObject)	\
+    ( (This)->lpVtbl -> QueryInterface(This,riid,ppvObject) ) 
+
+#define IChromeAccessibleDelegate_AddRef(This)	\
+    ( (This)->lpVtbl -> AddRef(This) ) 
+
+#define IChromeAccessibleDelegate_Release(This)	\
+    ( (This)->lpVtbl -> Release(This) ) 
+
+
+#define IChromeAccessibleDelegate_put_bulkFetchResult(This,requestID,resultJson)	\
+    ( (This)->lpVtbl -> put_bulkFetchResult(This,requestID,resultJson) ) 
+
+#define IChromeAccessibleDelegate_put_hitTestResult(This,requestID,result)	\
+    ( (This)->lpVtbl -> put_hitTestResult(This,requestID,result) ) 
+
+#endif /* COBJMACROS */
+
+
+#endif 	/* C style interface */
+
+
+
+
+#endif 	/* __IChromeAccessibleDelegate_INTERFACE_DEFINED__ */
+
+
+#ifndef __IChromeAccessible_INTERFACE_DEFINED__
+#define __IChromeAccessible_INTERFACE_DEFINED__
+
+/* interface IChromeAccessible */
+/* [unique][uuid][object] */ 
+
+
+EXTERN_C const IID IID_IChromeAccessible;
+
+#if defined(__cplusplus) && !defined(CINTERFACE)
+    
+    MIDL_INTERFACE("6175bd95-3b2e-4ebc-bc51-9cab782bec92")
+    IChromeAccessible : public IUnknown
+    {
+    public:
+        virtual /* [id][propget] */ HRESULT STDMETHODCALLTYPE get_bulkFetch( 
+            /* [in] */ BSTR inputJson,
+            /* [in] */ LONG requestID,
+            /* [in] */ IChromeAccessibleDelegate *delegate) = 0;
+        
+        virtual /* [id][propget] */ HRESULT STDMETHODCALLTYPE get_hitTest( 
+            /* [in] */ LONG screenPhysicalPixelX,
+            /* [in] */ LONG screenPhysicalPixelY,
+            /* [in] */ LONG requestID,
+            /* [in] */ IChromeAccessibleDelegate *delegate) = 0;
+        
+    };
+    
+    
+#else 	/* C style interface */
+
+    typedef struct IChromeAccessibleVtbl
+    {
+        BEGIN_INTERFACE
+        
+        HRESULT ( STDMETHODCALLTYPE *QueryInterface )( 
+            IChromeAccessible * This,
+            /* [in] */ REFIID riid,
+            /* [annotation][iid_is][out] */ 
+            _COM_Outptr_  void **ppvObject);
+        
+        ULONG ( STDMETHODCALLTYPE *AddRef )( 
+            IChromeAccessible * This);
+        
+        ULONG ( STDMETHODCALLTYPE *Release )( 
+            IChromeAccessible * This);
+        
+        /* [id][propget] */ HRESULT ( STDMETHODCALLTYPE *get_bulkFetch )( 
+            IChromeAccessible * This,
+            /* [in] */ BSTR inputJson,
+            /* [in] */ LONG requestID,
+            /* [in] */ IChromeAccessibleDelegate *delegate);
+        
+        /* [id][propget] */ HRESULT ( STDMETHODCALLTYPE *get_hitTest )( 
+            IChromeAccessible * This,
+            /* [in] */ LONG screenPhysicalPixelX,
+            /* [in] */ LONG screenPhysicalPixelY,
+            /* [in] */ LONG requestID,
+            /* [in] */ IChromeAccessibleDelegate *delegate);
+        
+        END_INTERFACE
+    } IChromeAccessibleVtbl;
+
+    interface IChromeAccessible
+    {
+        CONST_VTBL struct IChromeAccessibleVtbl *lpVtbl;
+    };
+
+    
+
+#ifdef COBJMACROS
+
+
+#define IChromeAccessible_QueryInterface(This,riid,ppvObject)	\
+    ( (This)->lpVtbl -> QueryInterface(This,riid,ppvObject) ) 
+
+#define IChromeAccessible_AddRef(This)	\
+    ( (This)->lpVtbl -> AddRef(This) ) 
+
+#define IChromeAccessible_Release(This)	\
+    ( (This)->lpVtbl -> Release(This) ) 
+
+
+#define IChromeAccessible_get_bulkFetch(This,inputJson,requestID,delegate)	\
+    ( (This)->lpVtbl -> get_bulkFetch(This,inputJson,requestID,delegate) ) 
+
+#define IChromeAccessible_get_hitTest(This,screenPhysicalPixelX,screenPhysicalPixelY,requestID,delegate)	\
+    ( (This)->lpVtbl -> get_hitTest(This,screenPhysicalPixelX,screenPhysicalPixelY,requestID,delegate) ) 
+
+#endif /* COBJMACROS */
+
+
+#endif 	/* C style interface */
+
+
+
+
+#endif 	/* __IChromeAccessible_INTERFACE_DEFINED__ */
+
+
+/* Additional Prototypes for ALL interfaces */
+
+unsigned long             __RPC_USER  BSTR_UserSize(     unsigned long *, unsigned long            , BSTR * ); 
+unsigned char * __RPC_USER  BSTR_UserMarshal(  unsigned long *, unsigned char *, BSTR * ); 
+unsigned char * __RPC_USER  BSTR_UserUnmarshal(unsigned long *, unsigned char *, BSTR * ); 
+void                      __RPC_USER  BSTR_UserFree(     unsigned long *, BSTR * ); 
+
+/* end of Additional Prototypes */
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif
+
+
diff --git a/third_party/win_build_output/midl/ui/accessibility/platform/arm64/ichromeaccessible_i.c b/third_party/win_build_output/midl/ui/accessibility/platform/arm64/ichromeaccessible_i.c
new file mode 100644
index 0000000000000000000000000000000000000000..3d818de2c4edb3b64ecbce847b2110619f59fa20
--- /dev/null
+++ b/third_party/win_build_output/midl/ui/accessibility/platform/arm64/ichromeaccessible_i.c
@@ -0,0 +1,82 @@
+
+
+/* this ALWAYS GENERATED file contains the IIDs and CLSIDs */
+
+/* link this file in with the server and any clients */
+
+
+ /* File created by MIDL compiler version 8.xx.xxxx */
+/* at a redacted point in time
+ */
+/* Compiler settings for ../../ui/accessibility/platform/ichromeaccessible.idl:
+    Oicf, W1, Zp8, env=Win64 (32b run), target_arch=ARM64 8.01.0622 
+    protocol : dce , ms_ext, c_ext, robust
+    error checks: allocation ref bounds_check enum stub_data 
+    VC __declspec() decoration level: 
+         __declspec(uuid()), __declspec(selectany), __declspec(novtable)
+         DECLSPEC_UUID(), MIDL_INTERFACE()
+*/
+/* @@MIDL_FILE_HEADING(  ) */
+
+#pragma warning( disable: 4049 )  /* more than 64k source lines */
+
+
+#ifdef __cplusplus
+extern "C"{
+#endif 
+
+
+#include <rpc.h>
+#include <rpcndr.h>
+
+#ifdef _MIDL_USE_GUIDDEF_
+
+#ifndef INITGUID
+#define INITGUID
+#include <guiddef.h>
+#undef INITGUID
+#else
+#include <guiddef.h>
+#endif
+
+#define MIDL_DEFINE_GUID(type,name,l,w1,w2,b1,b2,b3,b4,b5,b6,b7,b8) \
+        DEFINE_GUID(name,l,w1,w2,b1,b2,b3,b4,b5,b6,b7,b8)
+
+#else // !_MIDL_USE_GUIDDEF_
+
+#ifndef __IID_DEFINED__
+#define __IID_DEFINED__
+
+typedef struct _IID
+{
+    unsigned long x;
+    unsigned short s1;
+    unsigned short s2;
+    unsigned char  c[8];
+} IID;
+
+#endif // __IID_DEFINED__
+
+#ifndef CLSID_DEFINED
+#define CLSID_DEFINED
+typedef IID CLSID;
+#endif // CLSID_DEFINED
+
+#define MIDL_DEFINE_GUID(type,name,l,w1,w2,b1,b2,b3,b4,b5,b6,b7,b8) \
+        EXTERN_C __declspec(selectany) const type name = {l,w1,w2,{b1,b2,b3,b4,b5,b6,b7,b8}}
+
+#endif // !_MIDL_USE_GUIDDEF_
+
+MIDL_DEFINE_GUID(IID, IID_IChromeAccessibleDelegate,0x0e3edc14,0x79f4,0x413f,0xb8,0x54,0xd3,0xb6,0x86,0x0d,0x74,0xa2);
+
+
+MIDL_DEFINE_GUID(IID, IID_IChromeAccessible,0x6175bd95,0x3b2e,0x4ebc,0xbc,0x51,0x9c,0xab,0x78,0x2b,0xec,0x92);
+
+#undef MIDL_DEFINE_GUID
+
+#ifdef __cplusplus
+}
+#endif
+
+
+
diff --git a/third_party/win_build_output/midl/ui/accessibility/platform/arm64/ichromeaccessible_p.c b/third_party/win_build_output/midl/ui/accessibility/platform/arm64/ichromeaccessible_p.c
new file mode 100644
index 0000000000000000000000000000000000000000..e369303de99f831a23ff794e54ad9e5feae935eb
--- /dev/null
+++ b/third_party/win_build_output/midl/ui/accessibility/platform/arm64/ichromeaccessible_p.c
@@ -0,0 +1,572 @@
+
+
+/* this ALWAYS GENERATED file contains the proxy stub code */
+
+
+ /* File created by MIDL compiler version 8.xx.xxxx */
+/* at a redacted point in time
+ */
+/* Compiler settings for ../../ui/accessibility/platform/ichromeaccessible.idl:
+    Oicf, W1, Zp8, env=Win64 (32b run), target_arch=ARM64 8.01.0622 
+    protocol : dce , ms_ext, c_ext, robust
+    error checks: allocation ref bounds_check enum stub_data 
+    VC __declspec() decoration level: 
+         __declspec(uuid()), __declspec(selectany), __declspec(novtable)
+         DECLSPEC_UUID(), MIDL_INTERFACE()
+*/
+/* @@MIDL_FILE_HEADING(  ) */
+
+#if defined(_M_ARM64)
+
+
+#pragma warning( disable: 4049 )  /* more than 64k source lines */
+#if _MSC_VER >= 1200
+#pragma warning(push)
+#endif
+
+#pragma warning( disable: 4211 )  /* redefine extern to static */
+#pragma warning( disable: 4232 )  /* dllimport identity*/
+#pragma warning( disable: 4024 )  /* array to pointer mapping*/
+#pragma warning( disable: 4152 )  /* function/data pointer conversion in expression */
+
+#define USE_STUBLESS_PROXY
+
+
+/* verify that the <rpcproxy.h> version is high enough to compile this file*/
+#ifndef __REDQ_RPCPROXY_H_VERSION__
+#define __REQUIRED_RPCPROXY_H_VERSION__ 475
+#endif
+
+
+#include "rpcproxy.h"
+#ifndef __RPCPROXY_H_VERSION__
+#error this stub requires an updated version of <rpcproxy.h>
+#endif /* __RPCPROXY_H_VERSION__ */
+
+
+#include "ichromeaccessible.h"
+
+#define TYPE_FORMAT_STRING_SIZE   75                                
+#define PROC_FORMAT_STRING_SIZE   215                               
+#define EXPR_FORMAT_STRING_SIZE   1                                 
+#define TRANSMIT_AS_TABLE_SIZE    0            
+#define WIRE_MARSHAL_TABLE_SIZE   1            
+
+typedef struct _ichromeaccessible_MIDL_TYPE_FORMAT_STRING
+    {
+    short          Pad;
+    unsigned char  Format[ TYPE_FORMAT_STRING_SIZE ];
+    } ichromeaccessible_MIDL_TYPE_FORMAT_STRING;
+
+typedef struct _ichromeaccessible_MIDL_PROC_FORMAT_STRING
+    {
+    short          Pad;
+    unsigned char  Format[ PROC_FORMAT_STRING_SIZE ];
+    } ichromeaccessible_MIDL_PROC_FORMAT_STRING;
+
+typedef struct _ichromeaccessible_MIDL_EXPR_FORMAT_STRING
+    {
+    long          Pad;
+    unsigned char  Format[ EXPR_FORMAT_STRING_SIZE ];
+    } ichromeaccessible_MIDL_EXPR_FORMAT_STRING;
+
+
+static const RPC_SYNTAX_IDENTIFIER  _RpcTransferSyntax = 
+{{0x8A885D04,0x1CEB,0x11C9,{0x9F,0xE8,0x08,0x00,0x2B,0x10,0x48,0x60}},{2,0}};
+
+
+extern const ichromeaccessible_MIDL_TYPE_FORMAT_STRING ichromeaccessible__MIDL_TypeFormatString;
+extern const ichromeaccessible_MIDL_PROC_FORMAT_STRING ichromeaccessible__MIDL_ProcFormatString;
+extern const ichromeaccessible_MIDL_EXPR_FORMAT_STRING ichromeaccessible__MIDL_ExprFormatString;
+
+
+extern const MIDL_STUB_DESC Object_StubDesc;
+
+
+extern const MIDL_SERVER_INFO IChromeAccessibleDelegate_ServerInfo;
+extern const MIDL_STUBLESS_PROXY_INFO IChromeAccessibleDelegate_ProxyInfo;
+
+
+extern const MIDL_STUB_DESC Object_StubDesc;
+
+
+extern const MIDL_SERVER_INFO IChromeAccessible_ServerInfo;
+extern const MIDL_STUBLESS_PROXY_INFO IChromeAccessible_ProxyInfo;
+
+
+extern const USER_MARSHAL_ROUTINE_QUADRUPLE UserMarshalRoutines[ WIRE_MARSHAL_TABLE_SIZE ];
+
+#if !defined(__RPC_ARM64__)
+#error  Invalid build platform for this stub.
+#endif
+
+static const ichromeaccessible_MIDL_PROC_FORMAT_STRING ichromeaccessible__MIDL_ProcFormatString =
+    {
+        0,
+        {
+
+	/* Procedure put_bulkFetchResult */
+
+			0x33,		/* FC_AUTO_HANDLE */
+			0x6c,		/* Old Flags:  object, Oi2 */
+/*  2 */	NdrFcLong( 0x0 ),	/* 0 */
+/*  6 */	NdrFcShort( 0x3 ),	/* 3 */
+/*  8 */	NdrFcShort( 0x20 ),	/* ARM64 Stack size/offset = 32 */
+/* 10 */	NdrFcShort( 0x8 ),	/* 8 */
+/* 12 */	NdrFcShort( 0x8 ),	/* 8 */
+/* 14 */	0x46,		/* Oi2 Flags:  clt must size, has return, has ext, */
+			0x3,		/* 3 */
+/* 16 */	0xe,		/* 14 */
+			0x5,		/* Ext Flags:  new corr desc, srv corr check, */
+/* 18 */	NdrFcShort( 0x0 ),	/* 0 */
+/* 20 */	NdrFcShort( 0x1 ),	/* 1 */
+/* 22 */	NdrFcShort( 0x0 ),	/* 0 */
+/* 24 */	NdrFcShort( 0x3 ),	/* 3 */
+/* 26 */	0x3,		/* 3 */
+			0x80,		/* 128 */
+/* 28 */	0x81,		/* 129 */
+			0x82,		/* 130 */
+
+	/* Parameter requestID */
+
+/* 30 */	NdrFcShort( 0x48 ),	/* Flags:  in, base type, */
+/* 32 */	NdrFcShort( 0x8 ),	/* ARM64 Stack size/offset = 8 */
+/* 34 */	0x8,		/* FC_LONG */
+			0x0,		/* 0 */
+
+	/* Parameter resultJson */
+
+/* 36 */	NdrFcShort( 0x8b ),	/* Flags:  must size, must free, in, by val, */
+/* 38 */	NdrFcShort( 0x10 ),	/* ARM64 Stack size/offset = 16 */
+/* 40 */	NdrFcShort( 0x1c ),	/* Type Offset=28 */
+
+	/* Return value */
+
+/* 42 */	NdrFcShort( 0x70 ),	/* Flags:  out, return, base type, */
+/* 44 */	NdrFcShort( 0x18 ),	/* ARM64 Stack size/offset = 24 */
+/* 46 */	0x8,		/* FC_LONG */
+			0x0,		/* 0 */
+
+	/* Procedure put_hitTestResult */
+
+/* 48 */	0x33,		/* FC_AUTO_HANDLE */
+			0x6c,		/* Old Flags:  object, Oi2 */
+/* 50 */	NdrFcLong( 0x0 ),	/* 0 */
+/* 54 */	NdrFcShort( 0x4 ),	/* 4 */
+/* 56 */	NdrFcShort( 0x20 ),	/* ARM64 Stack size/offset = 32 */
+/* 58 */	NdrFcShort( 0x8 ),	/* 8 */
+/* 60 */	NdrFcShort( 0x8 ),	/* 8 */
+/* 62 */	0x46,		/* Oi2 Flags:  clt must size, has return, has ext, */
+			0x3,		/* 3 */
+/* 64 */	0xe,		/* 14 */
+			0x1,		/* Ext Flags:  new corr desc, */
+/* 66 */	NdrFcShort( 0x0 ),	/* 0 */
+/* 68 */	NdrFcShort( 0x0 ),	/* 0 */
+/* 70 */	NdrFcShort( 0x0 ),	/* 0 */
+/* 72 */	NdrFcShort( 0x3 ),	/* 3 */
+/* 74 */	0x3,		/* 3 */
+			0x80,		/* 128 */
+/* 76 */	0x81,		/* 129 */
+			0x82,		/* 130 */
+
+	/* Parameter requestID */
+
+/* 78 */	NdrFcShort( 0x48 ),	/* Flags:  in, base type, */
+/* 80 */	NdrFcShort( 0x8 ),	/* ARM64 Stack size/offset = 8 */
+/* 82 */	0x8,		/* FC_LONG */
+			0x0,		/* 0 */
+
+	/* Parameter result */
+
+/* 84 */	NdrFcShort( 0xb ),	/* Flags:  must size, must free, in, */
+/* 86 */	NdrFcShort( 0x10 ),	/* ARM64 Stack size/offset = 16 */
+/* 88 */	NdrFcShort( 0x26 ),	/* Type Offset=38 */
+
+	/* Return value */
+
+/* 90 */	NdrFcShort( 0x70 ),	/* Flags:  out, return, base type, */
+/* 92 */	NdrFcShort( 0x18 ),	/* ARM64 Stack size/offset = 24 */
+/* 94 */	0x8,		/* FC_LONG */
+			0x0,		/* 0 */
+
+	/* Procedure get_bulkFetch */
+
+/* 96 */	0x33,		/* FC_AUTO_HANDLE */
+			0x6c,		/* Old Flags:  object, Oi2 */
+/* 98 */	NdrFcLong( 0x0 ),	/* 0 */
+/* 102 */	NdrFcShort( 0x3 ),	/* 3 */
+/* 104 */	NdrFcShort( 0x28 ),	/* ARM64 Stack size/offset = 40 */
+/* 106 */	NdrFcShort( 0x8 ),	/* 8 */
+/* 108 */	NdrFcShort( 0x8 ),	/* 8 */
+/* 110 */	0x46,		/* Oi2 Flags:  clt must size, has return, has ext, */
+			0x4,		/* 4 */
+/* 112 */	0x10,		/* 16 */
+			0x5,		/* Ext Flags:  new corr desc, srv corr check, */
+/* 114 */	NdrFcShort( 0x0 ),	/* 0 */
+/* 116 */	NdrFcShort( 0x1 ),	/* 1 */
+/* 118 */	NdrFcShort( 0x0 ),	/* 0 */
+/* 120 */	NdrFcShort( 0x4 ),	/* 4 */
+/* 122 */	0x4,		/* 4 */
+			0x80,		/* 128 */
+/* 124 */	0x81,		/* 129 */
+			0x82,		/* 130 */
+/* 126 */	0x83,		/* 131 */
+			0x0,		/* 0 */
+
+	/* Parameter inputJson */
+
+/* 128 */	NdrFcShort( 0x8b ),	/* Flags:  must size, must free, in, by val, */
+/* 130 */	NdrFcShort( 0x8 ),	/* ARM64 Stack size/offset = 8 */
+/* 132 */	NdrFcShort( 0x1c ),	/* Type Offset=28 */
+
+	/* Parameter requestID */
+
+/* 134 */	NdrFcShort( 0x48 ),	/* Flags:  in, base type, */
+/* 136 */	NdrFcShort( 0x10 ),	/* ARM64 Stack size/offset = 16 */
+/* 138 */	0x8,		/* FC_LONG */
+			0x0,		/* 0 */
+
+	/* Parameter delegate */
+
+/* 140 */	NdrFcShort( 0xb ),	/* Flags:  must size, must free, in, */
+/* 142 */	NdrFcShort( 0x18 ),	/* ARM64 Stack size/offset = 24 */
+/* 144 */	NdrFcShort( 0x38 ),	/* Type Offset=56 */
+
+	/* Return value */
+
+/* 146 */	NdrFcShort( 0x70 ),	/* Flags:  out, return, base type, */
+/* 148 */	NdrFcShort( 0x20 ),	/* ARM64 Stack size/offset = 32 */
+/* 150 */	0x8,		/* FC_LONG */
+			0x0,		/* 0 */
+
+	/* Procedure get_hitTest */
+
+/* 152 */	0x33,		/* FC_AUTO_HANDLE */
+			0x6c,		/* Old Flags:  object, Oi2 */
+/* 154 */	NdrFcLong( 0x0 ),	/* 0 */
+/* 158 */	NdrFcShort( 0x4 ),	/* 4 */
+/* 160 */	NdrFcShort( 0x30 ),	/* ARM64 Stack size/offset = 48 */
+/* 162 */	NdrFcShort( 0x18 ),	/* 24 */
+/* 164 */	NdrFcShort( 0x8 ),	/* 8 */
+/* 166 */	0x46,		/* Oi2 Flags:  clt must size, has return, has ext, */
+			0x5,		/* 5 */
+/* 168 */	0x10,		/* 16 */
+			0x1,		/* Ext Flags:  new corr desc, */
+/* 170 */	NdrFcShort( 0x0 ),	/* 0 */
+/* 172 */	NdrFcShort( 0x0 ),	/* 0 */
+/* 174 */	NdrFcShort( 0x0 ),	/* 0 */
+/* 176 */	NdrFcShort( 0x5 ),	/* 5 */
+/* 178 */	0x5,		/* 5 */
+			0x80,		/* 128 */
+/* 180 */	0x81,		/* 129 */
+			0x82,		/* 130 */
+/* 182 */	0x83,		/* 131 */
+			0x84,		/* 132 */
+
+	/* Parameter screenPhysicalPixelX */
+
+/* 184 */	NdrFcShort( 0x48 ),	/* Flags:  in, base type, */
+/* 186 */	NdrFcShort( 0x8 ),	/* ARM64 Stack size/offset = 8 */
+/* 188 */	0x8,		/* FC_LONG */
+			0x0,		/* 0 */
+
+	/* Parameter screenPhysicalPixelY */
+
+/* 190 */	NdrFcShort( 0x48 ),	/* Flags:  in, base type, */
+/* 192 */	NdrFcShort( 0x10 ),	/* ARM64 Stack size/offset = 16 */
+/* 194 */	0x8,		/* FC_LONG */
+			0x0,		/* 0 */
+
+	/* Parameter requestID */
+
+/* 196 */	NdrFcShort( 0x48 ),	/* Flags:  in, base type, */
+/* 198 */	NdrFcShort( 0x18 ),	/* ARM64 Stack size/offset = 24 */
+/* 200 */	0x8,		/* FC_LONG */
+			0x0,		/* 0 */
+
+	/* Parameter delegate */
+
+/* 202 */	NdrFcShort( 0xb ),	/* Flags:  must size, must free, in, */
+/* 204 */	NdrFcShort( 0x20 ),	/* ARM64 Stack size/offset = 32 */
+/* 206 */	NdrFcShort( 0x38 ),	/* Type Offset=56 */
+
+	/* Return value */
+
+/* 208 */	NdrFcShort( 0x70 ),	/* Flags:  out, return, base type, */
+/* 210 */	NdrFcShort( 0x28 ),	/* ARM64 Stack size/offset = 40 */
+/* 212 */	0x8,		/* FC_LONG */
+			0x0,		/* 0 */
+
+			0x0
+        }
+    };
+
+static const ichromeaccessible_MIDL_TYPE_FORMAT_STRING ichromeaccessible__MIDL_TypeFormatString =
+    {
+        0,
+        {
+			NdrFcShort( 0x0 ),	/* 0 */
+/*  2 */	
+			0x12, 0x0,	/* FC_UP */
+/*  4 */	NdrFcShort( 0xe ),	/* Offset= 14 (18) */
+/*  6 */	
+			0x1b,		/* FC_CARRAY */
+			0x1,		/* 1 */
+/*  8 */	NdrFcShort( 0x2 ),	/* 2 */
+/* 10 */	0x9,		/* Corr desc: FC_ULONG */
+			0x0,		/*  */
+/* 12 */	NdrFcShort( 0xfffc ),	/* -4 */
+/* 14 */	NdrFcShort( 0x1 ),	/* Corr flags:  early, */
+/* 16 */	0x6,		/* FC_SHORT */
+			0x5b,		/* FC_END */
+/* 18 */	
+			0x17,		/* FC_CSTRUCT */
+			0x3,		/* 3 */
+/* 20 */	NdrFcShort( 0x8 ),	/* 8 */
+/* 22 */	NdrFcShort( 0xfff0 ),	/* Offset= -16 (6) */
+/* 24 */	0x8,		/* FC_LONG */
+			0x8,		/* FC_LONG */
+/* 26 */	0x5c,		/* FC_PAD */
+			0x5b,		/* FC_END */
+/* 28 */	0xb4,		/* FC_USER_MARSHAL */
+			0x83,		/* 131 */
+/* 30 */	NdrFcShort( 0x0 ),	/* 0 */
+/* 32 */	NdrFcShort( 0x8 ),	/* 8 */
+/* 34 */	NdrFcShort( 0x0 ),	/* 0 */
+/* 36 */	NdrFcShort( 0xffde ),	/* Offset= -34 (2) */
+/* 38 */	
+			0x2f,		/* FC_IP */
+			0x5a,		/* FC_CONSTANT_IID */
+/* 40 */	NdrFcLong( 0x0 ),	/* 0 */
+/* 44 */	NdrFcShort( 0x0 ),	/* 0 */
+/* 46 */	NdrFcShort( 0x0 ),	/* 0 */
+/* 48 */	0xc0,		/* 192 */
+			0x0,		/* 0 */
+/* 50 */	0x0,		/* 0 */
+			0x0,		/* 0 */
+/* 52 */	0x0,		/* 0 */
+			0x0,		/* 0 */
+/* 54 */	0x0,		/* 0 */
+			0x46,		/* 70 */
+/* 56 */	
+			0x2f,		/* FC_IP */
+			0x5a,		/* FC_CONSTANT_IID */
+/* 58 */	NdrFcLong( 0xe3edc14 ),	/* 239000596 */
+/* 62 */	NdrFcShort( 0x79f4 ),	/* 31220 */
+/* 64 */	NdrFcShort( 0x413f ),	/* 16703 */
+/* 66 */	0xb8,		/* 184 */
+			0x54,		/* 84 */
+/* 68 */	0xd3,		/* 211 */
+			0xb6,		/* 182 */
+/* 70 */	0x86,		/* 134 */
+			0xd,		/* 13 */
+/* 72 */	0x74,		/* 116 */
+			0xa2,		/* 162 */
+
+			0x0
+        }
+    };
+
+static const USER_MARSHAL_ROUTINE_QUADRUPLE UserMarshalRoutines[ WIRE_MARSHAL_TABLE_SIZE ] = 
+        {
+            
+            {
+            BSTR_UserSize
+            ,BSTR_UserMarshal
+            ,BSTR_UserUnmarshal
+            ,BSTR_UserFree
+            }
+
+        };
+
+
+
+/* Standard interface: __MIDL_itf_ichromeaccessible_0000_0000, ver. 0.0,
+   GUID={0x00000000,0x0000,0x0000,{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00}} */
+
+
+/* Object interface: IUnknown, ver. 0.0,
+   GUID={0x00000000,0x0000,0x0000,{0xC0,0x00,0x00,0x00,0x00,0x00,0x00,0x46}} */
+
+
+/* Object interface: IChromeAccessibleDelegate, ver. 0.0,
+   GUID={0x0e3edc14,0x79f4,0x413f,{0xb8,0x54,0xd3,0xb6,0x86,0x0d,0x74,0xa2}} */
+
+#pragma code_seg(".orpc")
+static const unsigned short IChromeAccessibleDelegate_FormatStringOffsetTable[] =
+    {
+    0,
+    48
+    };
+
+static const MIDL_STUBLESS_PROXY_INFO IChromeAccessibleDelegate_ProxyInfo =
+    {
+    &Object_StubDesc,
+    ichromeaccessible__MIDL_ProcFormatString.Format,
+    &IChromeAccessibleDelegate_FormatStringOffsetTable[-3],
+    0,
+    0,
+    0
+    };
+
+
+static const MIDL_SERVER_INFO IChromeAccessibleDelegate_ServerInfo = 
+    {
+    &Object_StubDesc,
+    0,
+    ichromeaccessible__MIDL_ProcFormatString.Format,
+    &IChromeAccessibleDelegate_FormatStringOffsetTable[-3],
+    0,
+    0,
+    0,
+    0};
+CINTERFACE_PROXY_VTABLE(5) _IChromeAccessibleDelegateProxyVtbl = 
+{
+    &IChromeAccessibleDelegate_ProxyInfo,
+    &IID_IChromeAccessibleDelegate,
+    IUnknown_QueryInterface_Proxy,
+    IUnknown_AddRef_Proxy,
+    IUnknown_Release_Proxy ,
+    (void *) (INT_PTR) -1 /* IChromeAccessibleDelegate::put_bulkFetchResult */ ,
+    (void *) (INT_PTR) -1 /* IChromeAccessibleDelegate::put_hitTestResult */
+};
+
+const CInterfaceStubVtbl _IChromeAccessibleDelegateStubVtbl =
+{
+    &IID_IChromeAccessibleDelegate,
+    &IChromeAccessibleDelegate_ServerInfo,
+    5,
+    0, /* pure interpreted */
+    CStdStubBuffer_METHODS
+};
+
+
+/* Object interface: IChromeAccessible, ver. 0.0,
+   GUID={0x6175bd95,0x3b2e,0x4ebc,{0xbc,0x51,0x9c,0xab,0x78,0x2b,0xec,0x92}} */
+
+#pragma code_seg(".orpc")
+static const unsigned short IChromeAccessible_FormatStringOffsetTable[] =
+    {
+    96,
+    152
+    };
+
+static const MIDL_STUBLESS_PROXY_INFO IChromeAccessible_ProxyInfo =
+    {
+    &Object_StubDesc,
+    ichromeaccessible__MIDL_ProcFormatString.Format,
+    &IChromeAccessible_FormatStringOffsetTable[-3],
+    0,
+    0,
+    0
+    };
+
+
+static const MIDL_SERVER_INFO IChromeAccessible_ServerInfo = 
+    {
+    &Object_StubDesc,
+    0,
+    ichromeaccessible__MIDL_ProcFormatString.Format,
+    &IChromeAccessible_FormatStringOffsetTable[-3],
+    0,
+    0,
+    0,
+    0};
+CINTERFACE_PROXY_VTABLE(5) _IChromeAccessibleProxyVtbl = 
+{
+    &IChromeAccessible_ProxyInfo,
+    &IID_IChromeAccessible,
+    IUnknown_QueryInterface_Proxy,
+    IUnknown_AddRef_Proxy,
+    IUnknown_Release_Proxy ,
+    (void *) (INT_PTR) -1 /* IChromeAccessible::get_bulkFetch */ ,
+    (void *) (INT_PTR) -1 /* IChromeAccessible::get_hitTest */
+};
+
+const CInterfaceStubVtbl _IChromeAccessibleStubVtbl =
+{
+    &IID_IChromeAccessible,
+    &IChromeAccessible_ServerInfo,
+    5,
+    0, /* pure interpreted */
+    CStdStubBuffer_METHODS
+};
+
+static const MIDL_STUB_DESC Object_StubDesc = 
+    {
+    0,
+    NdrOleAllocate,
+    NdrOleFree,
+    0,
+    0,
+    0,
+    0,
+    0,
+    ichromeaccessible__MIDL_TypeFormatString.Format,
+    1, /* -error bounds_check flag */
+    0x50002, /* Ndr library version */
+    0,
+    0x801026e, /* MIDL Version 8.1.622 */
+    0,
+    UserMarshalRoutines,
+    0,  /* notify & notify_flag routine table */
+    0x1, /* MIDL flag */
+    0, /* cs routines */
+    0,   /* proxy/server info */
+    0
+    };
+
+const CInterfaceProxyVtbl * const _ichromeaccessible_ProxyVtblList[] = 
+{
+    ( CInterfaceProxyVtbl *) &_IChromeAccessibleDelegateProxyVtbl,
+    ( CInterfaceProxyVtbl *) &_IChromeAccessibleProxyVtbl,
+    0
+};
+
+const CInterfaceStubVtbl * const _ichromeaccessible_StubVtblList[] = 
+{
+    ( CInterfaceStubVtbl *) &_IChromeAccessibleDelegateStubVtbl,
+    ( CInterfaceStubVtbl *) &_IChromeAccessibleStubVtbl,
+    0
+};
+
+PCInterfaceName const _ichromeaccessible_InterfaceNamesList[] = 
+{
+    "IChromeAccessibleDelegate",
+    "IChromeAccessible",
+    0
+};
+
+
+#define _ichromeaccessible_CHECK_IID(n)	IID_GENERIC_CHECK_IID( _ichromeaccessible, pIID, n)
+
+int __stdcall _ichromeaccessible_IID_Lookup( const IID * pIID, int * pIndex )
+{
+    IID_BS_LOOKUP_SETUP
+
+    IID_BS_LOOKUP_INITIAL_TEST( _ichromeaccessible, 2, 1 )
+    IID_BS_LOOKUP_RETURN_RESULT( _ichromeaccessible, 2, *pIndex )
+    
+}
+
+const ExtendedProxyFileInfo ichromeaccessible_ProxyFileInfo = 
+{
+    (PCInterfaceProxyVtblList *) & _ichromeaccessible_ProxyVtblList,
+    (PCInterfaceStubVtblList *) & _ichromeaccessible_StubVtblList,
+    (const PCInterfaceName * ) & _ichromeaccessible_InterfaceNamesList,
+    0, /* no delegation */
+    & _ichromeaccessible_IID_Lookup, 
+    2,
+    2,
+    0, /* table of [async_uuid] interfaces */
+    0, /* Filler1 */
+    0, /* Filler2 */
+    0  /* Filler3 */
+};
+#if _MSC_VER >= 1200
+#pragma warning(pop)
+#endif
+
+
+#endif /* defined(_M_ARM64)*/
+
